client/js/nug-skeleton-app.js

```diff
@@ -1,24 +1,109 @@
   'use strict';
 
   var nugNgApp = angular.module('nug-skeleton-app', [
-    'ui.router'
+    'ui.router','auth0', 'angular-storage', 'angular-jwt'
   ]);
 
-  nugNgApp.config(function($stateProvider, $urlRouterProvider) {
+  nugNgApp.config(function($stateProvider, $urlRouterProvider, authProvider, $httpProvider, jwtInterceptorProvider) {
     $stateProvider
       .state('dashboard', {
         url: '/dashboard',
         templateUrl: 'partials/dashboard.html',
-        controller: 'dashboardCtrl'
+        controller: 'dashboardCtrl',
+        data: {
+     		requiresLogin: true
+   		}
       })
       .state('settings', {
         url: '/settings',
         templateUrl: 'partials/settings.html',
-        controller: 'settingsCtrl'
-      });
-  });
+        controller: 'settingsCtrl',
+         data: {
+     		requiresLogin: true
+   		}
+      })
+	  .state('login', {
+		url: '/login',
+		templateUrl: 'partials/login.html',
+		controller: 'loginCtrl'
+	  })
+	;
+  
+  	//used for Auth0
+  
+		authProvider.init({
+		domain: 'nugs.auth0.com',
+		clientID: 'TdtdYCDQHSR3TtNgsMuCXHfjHDyxMsmB',
```

Comment:
The clientID is paired with clientSecret.  Generally it seems like clientID is okay to show (per various examples) but clientSecret is not okay to share.  That said, probably better to hide both.  Do you have a design recommendation for that?  Save Node side?